src/components/home/header.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import logo from '../../images/logo_white.png';
+import userImage from '../../images/default-user-image.jpg';
```

Comment:
@noor-ahmed I think we can reduce more about size of this image 🙌
![Screen Shot 2563-11-18 at 15 29 04](https://user-images.githubusercontent.com/749672/99505049-e9e5c880-29b2-11eb-9799-6a017688c3ce.png)